about.html

```diff
@@ -0,0 +1,135 @@
+                    This year, we hope to expand our invitation list including organizations and individuals who want to
+                    work with us on
+                    shared projects that advance the cause of the Commons, free culture and open knowledge.
+                    So if you're active and engaged in the worlds of open content and knowledge - free software
+                    advocates, wikipedians, Open
+                    knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and
+                    activists - we hope you'll
+                    consider joining us this year to build a stronger, more vibrant commons together
+                </p>
+
+                <div class="abt-contact">
+                    <p class="abt-contact-desc">Please contact us per Email for nay further questions about CC Global
+                        Summit 2023!</p>
+                    <p class="about-contact-email">omololuayk@gmail.com</p>
+                </div>
+            </div>
+
+        </div>
+    </header>
+    <section class="abt-logo-section">
+        <div class="desktop-abt-logo">
+            <div class="abt-logo-text-container">
+                <h4 class="abt-logo-text">Creative Content Global Summit 2023 Logo</h4>
+                <span class="line"></span>
+                <p class="logo-section-desc">The logo of CC Global Summit 2023 was decided through the logo competition
+                    from 5, June to 7, July.</p>
+                <p class="desktop-logo-desc">The logo of CC Global Summit 2015 was decided through the logo competition
+                    from 8, June to 7, July.
+                    After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal
+                    won the prize.
+                    After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as
+                    follows:</p>
+            </div>
+            <img src="images/about_logo_desktop.png" class="abt-desktop-img" alt="gloal summit logo">
+        </div>
+    </section>
+    <section class="abt-past-summit">
+        <div class="abt-past-summit-content">
+            <h4 class="abt-logo-text">See the past CC Global submits</h4>
+            <span class="line"></span>
+            <p class="abt-past-summit-desc">Take a look at the past two CC Global Summits which took place in Buenos
+                Aires and in Warsaw</p>
+        </div>
+        <div class="abt-past-summit-pic">
+            <div class="abt-past-pic-edit">
+                <img src="images/about_2011 1.png" alt="global summit picture">
+            </div>
+            <div class="abt-past-pic-edit">
+                <img src="images/about_2013 1.png" alt="global summit picture">
+            </div>
+        </div>
+    </section>
+
+    <section id="Partner" class="partner-container abt-partner-container">
```

Comment:
- [ ] Your partners section on mobile devices doesn't have the appropriate spacing as seen in the provided image below, please kindly adjust your design.
- project design
![pt](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/e1abc84e-5263-4284-9737-828c8f27a4cb)
- your design
![pts](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/88ed19da-da3f-4f8d-9f8f-140b714c578b)